kategory-docs/docs/docs/optics/lens/README.md

```diff
@@ -149,4 +149,4 @@ firstTuple2<Int, String, String>().set(5 toT "World", "Hello, ")
 
 ### Laws
 
-Kategory provides [`LensLaws`](/docs/optics/laws#lenslaws) in the form of test cases for internal verification of lawful instances and third party apps creating their own lenses.
\ No newline at end of file
+Kategory provides [`LensLaws`]({{ '/docs/optics/laws#lenslaws' | relative_url }}) in the form of test cases for internal verification of lawful instances and third party apps creating their own lenses.
```

Comment:
Actually, this permalink does not exist, should I create it with an empty content?

Reply:
This can link to the sources where those laws are defined in the interim